specification/validated-relationships-service-api.yaml

```diff
@@ -280,6 +281,76 @@ paths:
+              | 400         | `INVALID_IDENTIFIER_SYSTEM`        | Invalid identifier system.                                                                                                                |
+              | 400         | `MISSING_IDENTIFIER_VALUE`         | Missing RelatedPerson NHS number.                                                                                                         |
+              | 400         | `INVALID_IDENTIFIER_VALUE`         | Malformed RelatedPerson NHS number.                                                                                                       |
+              | 400         | `INVALID_PATIENT_IDENTIFIER_VALUE` | Malformed Patient NHS number.                                                                                                             |
+              | 400         | `NOT_SUPPORTED`                    | The request is not currently supported.                                                                                                   |
+              | 401         | `ACCESS_DENIED`                    | Missing or invalid OAuth 2.0 bearer token in request.                                                                                     |
+              | 403         | `FORBIDDEN`                        | Access denied to resource.                                                                                                                |
+              | 404         | `INVALIDATED_RESOURCE`             | Resource that has been marked as invalid was requested - invalid resources cannot be retrieved                                            |
+              | 405         | `METHOD_NOT_ALLOWED`               | The method is not allowed.                                                                                                                |
+              | 408         | `TIMEOUT`                          | Request timed out.                                                                                                                        |
+              | 429         | `THROTTLED`                        | You have exceeded your application's [rate limit](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#rate-limits). |
+        
+            content:
+              application/fhir+json:
+                schema:
+                  $ref: '#/components/schemas/OperationOutcome'
+          "500":
+            description: Internal server error
+            content:
+              application/fhir+json:
+                schema:
+                  $ref: '#/components/schemas/OperationOutcome'
+                examples:  
+                  internalServerError:
+                    $ref: '#/components/examples/InternalServerError'
+
+
 components:
   schemas:
     QuestionnaireResponse:
```

Comment:
Going to put it here as it isn't highlighted by changes

There are missing data elements now due to further changes that have been made whilst you were doing a large refactor 

The main one is:
QuestionnaireResponseItem_RelatedPerson - line 564 
Would now sit around line 634

        {
          "linkId": "relatedPerson_relationshipMoreinfo",
          "text": "Relationship - further information",
          "answer": [
            {
              "valueString": "string"
            }
          ]
        }

Reply:
I think, due to the fairly drastic nature of the changes that you will probably need to look at all of the other changes that have been merged into master, update your branch, and then force a push of the whole file

Happy with the principal now of what you've implemented though!